release-notes/opensearch-dashboards.release-notes-2.2.0.md

```diff
@@ -0,0 +1,46 @@
+## Version 2.2.0 Release Notes
+
+#### Notable changes
+* Bump node version to 14.20.0 ([#2101](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2101))
+* OpenSearch Dashboards uses [OUI](https://github.com/opensearch-project/oui) and its alias onto EUI ([#2080](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2080))
+* New experimental feature: adds the Drag and Drop editor to Visualize. Note this is disabled by default. Please enable by setting wizard.enabled to true in opensearch_dashboards.yml ([#1966](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/1966))
```

Comment:
NIT: since markdown it could be:

`wizard.enabled: true` in `opensearch_dashboards.yml` but not required.

Reply:
I like the markdown. 

Also, there is another experimental flag that needs to be enabled, correct?

Reply:
I updated with the markdown
what experimental flag?